node/bundler.test.ts

```diff
@@ -536,3 +537,33 @@ test('Loads JSON modules', async () => {
   await cleanup()
   await rm(vendorDirectory.path, { force: true, recursive: true })
 })
+
+test('Supports TSX and process.env', async () => {
+  const { basePath, cleanup, distPath } = await useFixture('tsx')
+  const sourceDirectory = join(basePath, 'functions')
+  const declarations: Declaration[] = [
+    {
+      function: 'func1',
+      path: '/func1',
+    },
+  ]
+  const vendorDirectory = await tmp.dir()
+
+  await bundle([sourceDirectory], distPath, declarations, {
+    basePath,
+    vendorDirectory: vendorDirectory.path,
+  })
+
+  const manifestFile = await readFile(resolve(distPath, 'manifest.json'), 'utf8')
+  const manifest = JSON.parse(manifestFile)
+  const bundlePath = join(distPath, manifest.bundles[0].asset)
+  process.env.FOO = 'bar'
```

Comment:
Can we clean up after this?

Reply:
done in https://github.com/netlify/edge-bundler/pull/519/commits/5a9c7f3ec43f3e81c2fb3c11ce94edbe4f0fd3fc